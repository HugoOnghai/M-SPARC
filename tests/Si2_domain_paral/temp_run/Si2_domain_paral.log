
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Si2_domain_paral.inpt
# nprocs: 15
FD_ORDER: 12
BOUNDARY_CONDITION: 2
EXCHANGE_CORRELATION: GGA_PBE

TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8

ELEC_TEMP: 315.775131
ELEC_TEMP_TYPE: fd

NSTATES: 8
CALC_STRESS: 1

RELAX_MAXDILAT: 1.2
PRINT_ATOMS: 1
PRINT_FORCES: 1

CELL: 7.5 7.5 7.5   ## test comment
FD_GRID: 25 30 25         # FD grid

LATVEC:
0.5 0.5 0.0
0.0 0.5 0.5
0.5 0.0 0.5

KPOINT_GRID: 2 2 2


NP_BAND_PARAL: 3
NP_DOMAIN_PARAL: 3 5 1
<\INPT>

Warning: "BOUNDARY_CONDITION" is obsolete, use "BC" instead!
Neglecting option "NP_BAND_PARAL:", which is not supported in M-SPARC
Neglecting option "NP_DOMAIN_PARAL:", which is not supported in M-SPARC
 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 2
 Default atomic mass for Si is 28.085000
 pseudo_dir # 1 = Si.psp8

 COORD:
                   0                   0                   0
   1.875000000000000   1.875000000000000   1.875000000000000

   0.030168927102176

 kpoint grid before symmetry:
   0.209439510239320   0.209439510239320   0.209439510239320
  -0.209439510239320   0.209439510239320   0.209439510239320
   0.209439510239320  -0.209439510239320   0.209439510239320
  -0.209439510239320  -0.209439510239320   0.209439510239320
   0.209439510239320   0.209439510239320  -0.209439510239320
  -0.209439510239320   0.209439510239320  -0.209439510239320
   0.209439510239320  -0.209439510239320  -0.209439510239320
  -0.209439510239320  -0.209439510239320  -0.209439510239320

 kpoint grid after symmetry:
   0.209439510239320   0.209439510239320  -0.209439510239320
  -0.209439510239320   0.209439510239320  -0.209439510239320
   0.209439510239320  -0.209439510239320  -0.209439510239320
  -0.209439510239320  -0.209439510239320  -0.209439510239320

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 27
 Creating differentiation matrices ...
 Done. (4.010 sec)

 Estimated memory usage:
 Total:   88.98 MB
 orbitals            :   36.62 MB
 sparse matrices     :   48.35 MB
 global-size vectors :    2.00 MB
 mixing histories    :    2.00 MB

 Finding rb for Si ...
 rb = {3.900 3.900 3.900}, int_b = -4.000000436683456, err_rb = 4.367e-07
 rb = {4.200 4.200 4.200}, int_b = -4.000036573400556, err_rb = 3.657e-05
 rb = {4.500 4.500 4.500}, int_b = -4.000024269257735, err_rb = 2.427e-05
 rb = {4.800 4.800 4.800}, int_b = -4.000017997741731, err_rb = 1.800e-05
 rb = {5.100 5.100 5.100}, int_b = -4.000015178455217, err_rb = 1.518e-05
 rb = {5.400 5.400 5.400}, int_b = -4.000013138197377, err_rb = 1.314e-05
 rb = {5.700 5.700 5.700}, int_b = -4.000012725890136, err_rb = 1.273e-05
 rb = {6.000 6.000 6.000}, int_b = -4.000012122997310, err_rb = 1.212e-05
 rb = {6.300 6.300 6.300}, int_b = -4.000011874286570, err_rb = 1.187e-05
 rb = {6.600 6.600 6.600}, int_b = -4.000011119873331, err_rb = 1.112e-05
 rb = {6.900 6.900 6.900}, int_b = -4.000009851707460, err_rb = 9.852e-06
 rb = {7.200 7.200 7.200}, int_b = -4.000010245303357, err_rb = 1.025e-05
 rb = {7.500 7.500 7.500}, int_b = -4.000003414171170, err_rb = 3.414e-06
 rb = {7.800 7.800 7.800}, int_b = -3.999999730153339, err_rb = 2.698e-07
 rb = {8.100 8.100 8.100}, int_b = -4.000000056990022, err_rb = 5.699e-08
 rb = {8.400 8.400 8.400}, int_b = -3.999999987531885, err_rb = 1.247e-08
 rb = {8.700 8.700 8.700}, int_b = -3.999999999780883, err_rb = 2.191e-10
 rb = {8.700 8.700 8.700}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 8.000000002229

 Integration b_ref = 8.000000000009

 ****************************************
 *          Eself_ref = 57.509748       *
 ****************************************
 Done. (2.078058 s)
 Time for b calculation: 2.092 seconds.

 Starting calculating nonlocal projectors ...
 Done. (1.378583 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 1.41261e-09 in 37 iterations.
 Poisson problem took 1.474809s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 3.658700
 This SCF iteration took 21.827 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.553402
 This SCF iteration took 10.059 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.224805
 This SCF iteration took 9.852 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.211000
 AAR converged to a relative residual of 9.44687e-09 in 30 iterations.
 Poisson problem took 0.956891s
 Escc = 0.15961661
 Etot = -7.92418081
 Eatom = -3.96209041
 Error in SCF iteration: 2.5842e-02 
 This SCF iteration took 10.785 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.218268
 AAR converged to a relative residual of 3.65226e-09 in 25 iterations.
 Poisson problem took 0.792114s
 Escc = 0.04408317
 Etot = -7.93639203
 Eatom = -3.96819601
 Error in SCF iteration: 1.4069e-02 
 This SCF iteration took 10.600 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.218576
 AAR converged to a relative residual of 3.35569e-09 in 25 iterations.
 Poisson problem took 0.794529s
 Escc = -0.00278743
 Etot = -7.93785328
 Eatom = -3.96892664
 Error in SCF iteration: 3.9401e-03 
 This SCF iteration took 10.632 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.217750
 AAR converged to a relative residual of 8.69773e-09 in 21 iterations.
 Poisson problem took 0.683577s
 Escc = -0.00371577
 Etot = -7.93793171
 Eatom = -3.96896585
 Error in SCF iteration: 1.7649e-03 
 This SCF iteration took 10.467 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.217463
 AAR converged to a relative residual of 6.83513e-09 in 19 iterations.
 Poisson problem took 0.602524s
 Escc = -0.00030520
 Etot = -7.93793913
 Eatom = -3.96896956
 Error in SCF iteration: 3.6810e-04 
 This SCF iteration took 10.378 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.217460
 AAR converged to a relative residual of 9.45595e-09 in 15 iterations.
 Poisson problem took 0.471046s
 Escc = -0.00014024
 Etot = -7.93793978
 Eatom = -3.96896989
 Error in SCF iteration: 1.6904e-04 
 This SCF iteration took 10.907 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.217436
 AAR converged to a relative residual of 8.54213e-09 in 15 iterations.
 Poisson problem took 0.471866s
 Escc = 0.00008031
 Etot = -7.93793993
 Eatom = -3.96896996
 Error in SCF iteration: 5.1339e-05 
 This SCF iteration took 10.113 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.217447
 AAR converged to a relative residual of 6.6205e-09 in 13 iterations.
 Poisson problem took 0.409908s
 Escc = -0.00002638
 Etot = -7.93793993
 Eatom = -3.96896997
 Error in SCF iteration: 1.5940e-05 
 This SCF iteration took 9.983 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.217445
 AAR converged to a relative residual of 1.25085e-09 in 13 iterations.
 Poisson problem took 0.404306s
 Escc = 0.00001392
 Etot = -7.93793991
 Eatom = -3.96896996
 Error in SCF iteration: 5.4387e-06 
 This SCF iteration took 9.892 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.217447
 AAR converged to a relative residual of 1.01332e-09 in 13 iterations.
 Poisson problem took 0.411502s
 Escc = -0.00000360
 Etot = -7.93793992
 Eatom = -3.96896996
 Error in SCF iteration: 1.5963e-06 
 This SCF iteration took 9.869 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.217446
 AAR converged to a relative residual of 8.85837e-09 in 9 iterations.
 Poisson problem took 0.276036s
 Escc = 0.00000386
 Etot = -7.93793991
 Eatom = -3.96896996
 Error in SCF iteration: 1.0666e-06 
 This SCF iteration took 9.767 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = 0.217447
 AAR converged to a relative residual of 8.7255e-09 in 7 iterations.
 Poisson problem took 0.220204s
 Escc = -0.00000072
 Etot = -7.93793992
 Eatom = -3.96896996
 Error in SCF iteration: 1.9072e-07 
 This SCF iteration took 9.756 s.


 Finished SCF iteration in 12 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -7.937939924 Ha.       *
 *          Energy per atom = -3.968969962 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 2.159 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
   0.002443850166620   0.002443314007932   0.002443234309473
  -0.002443850166620  -0.002443314007932  -0.002443234309473

 Max magnitude of forces (Ha/Bohr):   0.004232207564526


 Time for calculating forces: 2.723882 s.

["Stress in GPa"


]   7.356271453802936   5.464851218686276   5.464876540522623
   5.464851218686276   7.394488292748442   5.482607210385525
   5.464876540522623   5.482607210385525   7.394485586316630


 Time for calculating stress: 8.217121 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.000000 	  0.000000 	  0.000000 
  1.875000 	  1.875000 	  1.875000 

 Run-time of the program: 186.323774 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.707106781186547
                        L1: 7.500000000000000
                        L2: 7.500000000000000
                        L3: 7.500000000000000
                        Nx: 25
                        Ny: 30
                        Nz: 25
                         N: 18750
                      ecut: -1
              mesh_spacing: -1
                   kptgrid: [4x3 double]
                  kptshift: [1x3 double]
                      nkpt: [2 2 2]
                     tnkpt: 4
                      wkpt: [4x1 double]
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 8
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157751310000000e+02
                       bet: 9.999996636976061e+02
                       npl: 27
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 8
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 7.848837209302326e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 1
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Si2_domain_paral'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x1 struct]
                     n_atm: 2
                     Atoms: [2x3 double]
           mvAtmConstraint: [2x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [2x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.300000000000000
                        dy: 0.250000000000000
                        dz: 0.300000000000000
                        dV: 0.015909902576697
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [18750x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [325x1 logical]
                 isOutr_11: [325x1 logical]
                      I_11: [325x1 double]
                     II_11: [325x1 double]
                      V_11: [325x1 double]
                 isOutl_22: [390x1 logical]
                 isOutr_22: [390x1 logical]
                      I_22: [390x1 double]
                     II_22: [390x1 double]
                      V_22: [390x1 double]
                 isOutl_33: [325x1 logical]
                 isOutr_33: [325x1 logical]
                      I_33: [325x1 double]
                     II_33: [325x1 double]
                      V_33: [325x1 double]
                  isOutl_1: [300x1 logical]
                  isOutr_1: [300x1 logical]
                       I_1: [300x1 double]
                      II_1: [300x1 double]
                       V_1: [300x1 double]
                  isOutl_2: [360x1 logical]
                  isOutr_2: [360x1 logical]
                       I_2: [360x1 double]
                      II_2: [360x1 double]
                       V_2: [360x1 double]
                  isOutl_3: [300x1 logical]
                  isOutr_3: [300x1 logical]
                       I_3: [300x1 double]
                      II_3: [300x1 double]
                       V_3: [300x1 double]
                 G_JOutl_1: [225000x1 logical]
                 G_JOutr_1: [225000x1 logical]
                     G_J_1: [225000x1 double]
                     G_I_1: [225000x1 double]
                     G_V_1: [225000x1 double]
                 G_JOutl_2: [225000x1 logical]
                 G_JOutr_2: [225000x1 logical]
                     G_J_2: [225000x1 double]
                     G_I_2: [225000x1 double]
                     G_V_2: [225000x1 double]
                 G_JOutl_3: [225000x1 logical]
                 G_JOutr_3: [225000x1 logical]
                     G_J_3: [225000x1 double]
                     G_I_3: [225000x1 double]
                     G_V_3: [225000x1 double]
                   Lap_std: [18750x18750 double]
                    grad_1: [18750x18750 double]
                    grad_2: [18750x18750 double]
                    grad_3: [18750x18750 double]
                LapPreconL: [18750x18750 double]
                LapPreconU: [18750x18750 double]
              memory_usage: 93300008
               staticfname: 'Si2_domain_paral.static'
                 suffixNum: 0
                  outfname: 'Si2_domain_paral.out'
                  parallel: 0
                         b: [18750x1 double]
                     b_ref: [18750x1 double]
                    rho_at: [18750x1 double]
                     Eself: 25.549300203303893
                 Eself_ref: 57.509748419210524
                       V_c: [18750x1 double]
                 PosCharge: 8.000000002229058
                 NegCharge: -8.000000002229058
                    E_corr: -3.710293363212713e-05
                       rho: [18750x1 double]
                      Atom: [1x2 struct]
                       phi: [18750x1 double]
                      e_xc: [18750x1 double]
                 dvxcdgrho: [18750x1 double]
                       Vxc: [18750x1 double]
                      Veff: [18750x1 double]
                         X: [18750x7 double]
                         F: [18750x7 double]
          mixing_hist_fkm1: [18750x1 double]
          mixing_hist_xkm1: [18750x1 double]
                       psi: [18750x8x4 double]
    upper_bound_guess_vecs: [18750x4 double]
                    EigVal: [8x4 double]
                  lambda_f: 0.217446772229548
                       occ: [8x4 double]
                    Etotal: -7.937939924085772
                     Eband: 0.027309015391983
                       Exc: -2.369643949323894
                    Exc_dc: -3.080817277235429
                  Eelec_dc: 16.872915760193912
                      Eent: -4.595246561306858e-11
                      Escc: -7.212997231353215e-07
                   S_Debug: [1x1 struct]
                     force: [2x3 double]
                 abs_force: [2x1 double]
                    Stress: [3x3 double]
                  Pressure: -7.381748444289336
             stress_dimred: [3x3 double]

