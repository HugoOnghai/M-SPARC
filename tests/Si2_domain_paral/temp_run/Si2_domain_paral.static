***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.0000000000       0.0000000000       0.0000000000
      0.2500000000       0.2500000000       0.2500000000
Atomic forces (Ha/Bohr):
      0.0024438502       0.0024433140       0.0024432343
     -0.0024438502      -0.0024433140      -0.0024432343
Stress (GPa):
  7.3562714538E+00   5.4648512187E+00   5.4648765405E+00 
  5.4648512187E+00   7.3944882927E+00   5.4826072104E+00 
  5.4648765405E+00   5.4826072104E+00   7.3944855863E+00 
