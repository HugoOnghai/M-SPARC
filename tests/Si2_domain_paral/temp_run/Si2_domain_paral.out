***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:41                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.500000 7.500000 7.500000 
LATVEC:
0.500000000000000 0.500000000000000 0.000000000000000 
0.000000000000000 0.500000000000000 0.500000000000000 
0.500000000000000 0.000000000000000 0.500000000000000 
FD_GRID: 25 30 25
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 5.000000e-01 5.000000e-01 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 27
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si2_domain_paral.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.300000 (Bohr)
Mesh spacing in y-direction        :  0.250000 (Bohr)
Mesh spacing in z direction        :  0.300000 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  Si2_domain_paral.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.70 8.70 8.70
Number of atoms of type 1          :  2
Estimated total memory usage       :  88.98 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9620904066E+00        2.584E-02        52.521
2            -3.9681960128E+00        1.407E-02        10.582
3            -3.9689266405E+00        3.940E-03        10.630
4            -3.9689658547E+00        1.765E-03        10.465
5            -3.9689695631E+00        3.681E-04        10.376
6            -3.9689698909E+00        1.690E-04        10.905
7            -3.9689699629E+00        5.134E-05        10.111
8            -3.9689699674E+00        1.594E-05        9.981
9            -3.9689699574E+00        5.439E-06        9.889
10           -3.9689699603E+00        1.596E-06        9.867
11           -3.9689699568E+00        1.067E-06        9.765
12           -3.9689699620E+00        1.907E-07        9.754
Total number of SCF: 12    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9689699620E+00 (Ha/atom)
Total free energy                  : -7.9379399241E+00 (Ha)
Band structure energy              :  2.7309015392E-02 (Ha)
Exchange correlation energy        : -2.3696439493E+00 (Ha)
Self and correction energy         : -2.5549337306E+01 (Ha)
Entropy*kb*T                       : -4.5952465613E-11 (Ha)
Fermi level                        :  2.1744677223E-01 (Ha)
Average force                      :  4.2322075645E-03 (Ha/Bohr)
Maximum force                      :  4.2322075645E-03 (Ha/Bohr)
Time for force calculation         :  2.724 (sec)
Pressure                           : -7.3817484443E+00 (GPa)
Max stress                         :  7.3944882927E+00 (GPa)
Time for stress calculation        :  8.219 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  186.324 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
